appcues/src/main/java/com/appcues/ui/primitive/SpacerPrimitive.kt

```diff
@@ -0,0 +1,11 @@
+package com.appcues.ui.primitive
+
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import com.appcues.data.model.ExperiencePrimitive.SpacerPrimitive
+
+@Composable
+internal fun SpacerPrimitive.Compose(modifier: Modifier) {
+    Spacer(modifier = modifier)
```

Comment:
it could be any Composition, or none really. it doesnt actually make any difference but I think this keeps it at least in pair with other primitive types.